webrtc-stats.html

```diff
@@ -2049,6 +2050,21 @@ <h2>
                   MediaStreamTrack, if <code>type</code> is <code>"track"</code>).
                 </p>
               </dd>
+              <dt>
+                <dfn><code>hugeFramesSent</code></dfn> of type <span class="idlMemberType">
+                <a>unsigned long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Represents the total number of huge frames sent by this RTCRtpSender (or for this
+                  MediaStreamTrack, if <code>type</code> is <code>"track"</code>). Huge frames, by
+                  definition, are frames with encoded size at least 2.5 times the target bitrate
+                  per second divided by target fps set for the encoder at the time the frame was
+                  encoded. These are usually difficult to encode frames with a lot of changes in the
+                  picture. This can be used to estimate, e.g slide changes in the streamed
```

Comment:
These are usually difficult to encode frames with a lot of changes in the picture --> not sure if I understand this correctly, if there are a lot of changes in the picture, would it not generate a Key Frame?

Reply:
This metric should also state if KeyFrames are excluded or included?

Reply:
I only considered to add this metric here, because this is what reviewers of webrtc changelist requested.

Usually the encoders don't produce keyframes, even if there are a lot of changes in the frame. 

This is why kayFramesSent can't be used for the purpose of detecting slide changes. Thus, the huge frames concept. It's entirely separate from keyframes. While key frames most probably would be counted in the metric, they are not guaranteed to be so.